tests/ts_A_select_compare_two_fields_not_allowed.erl

```diff
@@ -39,7 +39,6 @@ confirm() ->
         "AND myfamily = 'fa2mily1' "
         "AND myseries ='seriesX' "
         "AND weather = myseries",
-    Expected = "Expect that fields cannot be compared",
-    Got = ts_util:ts_query(ts_util:cluster_and_connect(single), normal, DDL, Data, Qry),
-    ?assertEqual(Expected, Got),
+    {error, Got} = ts_util:ts_query(ts_util:cluster_and_connect(single), normal, DDL, Data, Qry),
+    ?assertNotEqual(0, string:str(binary_to_list(Got), "Comparing or otherwise operating on two fields is not supported")),
```

Comment:
Really? I just refactored the test to avoid that case. Why would you like to make it more brittle when we are (hopefully) going to change the code/message?